SWLOR.Game.Server/Feature/SpawnDefinition/HutlarSpawnDefinition.cs

```diff
@@ -112,5 +113,50 @@ private void QionFoothills()
                 .RandomlyWalks()
                 .ReturnsHome();
         }
+        private void QionHive()
+        {
+            _builder.Create("HutlarDungeonBroodmother")
```

Comment:
Please follow convention by capitalizing all loot table names/tags. HutlarDungeonBroodmother -> HUTLAR_DUNGEON_BROODMOTHER as an example. Update any references on other objects to match.